### PR TITLE
[APPEND][FSTORE-709] add stream fg key to cached_feature_extra_constraints

### PR DESCRIPTION
**Pull request by kennethmhc**

### CODE REVIEW

File: files/default/sql/ddl/updates/3.3.0.sql [hunk 1]
```diff
@@ -145,6 +145,10 @@ ALTER TABLE `hopsworks`.`hdfs_command_execution` ADD UNIQUE KEY `uq_src_path` (`
 ALTER TABLE `hopsworks`.`feature_store_s3_connector` ADD COLUMN `arguments` VARCHAR(2000) DEFAULT NULL;
 
 --FSTORE-709
+DELETE FROM `cached_feature_extra_constraints` WHERE `stream_feature_group_id` NOT IN (
+    SELECT `id`
+    FROM `stream_feature_group`
+);
```
Comment (SirOibaf) [inline files/default/sql/ddl/updates/3.3.0.sql:151]: I have a feeling that this is not going to work on app. I think there will be too many constraints left pending.

Comment (kennethmhc) [reply files/default/sql/ddl/updates/3.3.0.sql:151]: @SirOibaf Do you mean 
```
    SELECT `id`
    FROM `stream_feature_group`
```
has too many record, and it won't work?

Comment (SirOibaf) [reply files/default/sql/ddl/updates/3.3.0.sql:151]: Yeah, one delete transaction can delete up to 32K records. I don't know how many contraints are pending on app.

**Comment by SirOibaf** [comment]
This PR is superseeded by this one: https://github.com/logicalclocks/hopsworks-chef/pull/980

**Pull request closed without merging**